### PR TITLE
Add Opsdroid info 

### PR DESCRIPTION
**Pull request by mohatagarvit**
Opsdroid info for giving swag has been added to the companies.

### CODE REVIEW

**Comment by crweiner** [comment]
Hay @mohatagarvit, thanks for taking the time to add Opsdroid!

Instead of the long list for **Requirements**, how about we change it to something like "perform at least one of the actions mentioned in the blog post."?

I know you've been fighting CodeClimate and I'm sorry to negate all of that hard work, but I think a one-liner like the rest of the items on the list is much easier to read.

Thanks!

**Comment by mohatagarvit** [comment]
Thanks @crweiner ! I was thinking the same that it was getting a little cumbersome. 
I have only given the link for the blog post in the Note section. Should I also attach the same link in the requirements?

File: README.md [hunk 1]
```diff
@@ -78,6 +87,15 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **How to sign up**: Make a PR on [Appwrite Github Repo](https://github.com/appwrite/appwrite) and we'll send you a link.
 - **Notes**: See the [blog post](https://medium.com/appwrite-io/hacktoberfest-2019-is-almost-here-lets-celebrate-it-together-24b311236dd?source=friends_link&sk=104527c24e85006a997fa068bfd1760d) for more information
 
+### O
```
Comment (crweiner) [inline README.md]: Don't need the "O" header here.

File: README.md [hunk 2]
```diff
@@ -47,6 +47,15 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **How to sign up**: [Join #Orbtoberfest](https://circleci-public.github.io/hacktoberfest/#/) and submit your PRs to this [Google Form](https://docs.google.com/forms/d/1-MbzLy0m1eRLjeye9wu76fYkzcX5A3PvgKd09EhVObY/viewform).
 - **Notes**: View full details above. Make pull requests to any open-source "orbs" for CircleCI, or create your own. When complete submit PRs to the Google form to claim swag.
 
+### O
+
+#### **Opsdroid**
+
+- **Swag**: Stickers, Limited-Edition Stickers
+- **Requirements**: Perform atleast one of the actions mentioned in the blog post
```
Comment (crweiner) [inline README.md]: Double-check spelling, please.

File: README.md [hunk 3]
```diff
@@ -78,6 +87,15 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **How to sign up**: Make a PR on [Appwrite Github Repo](https://github.com/appwrite/appwrite) and we'll send you a link.
 - **Notes**: See the [blog post](https://medium.com/appwrite-io/hacktoberfest-2019-is-almost-here-lets-celebrate-it-together-24b311236dd?source=friends_link&sk=104527c24e85006a997fa068bfd1760d) for more information
 
+### O
+
+#### **Opsdroid**
+
+- **Swag**: Stickers, Limited-Edition Stickers
+- **Requirements**: Perform atleast one of the actions mentioned in the blog post
```
Comment (crweiner) [inline README.md]: Double-check your spelling here as well, please.

**Comment by crweiner** [comment]
> Thanks @crweiner ! I was thinking the same that it was getting a little cumbersome.
> I have only given the link for the blog post in the Note section. Should I also attach the same link in the requirements?

That's definitely much better as a single line. No need to add the same link twice. :smile: 

If you could please just take another glance at everything, tighty up the few things I mentioned in the review comments above, and resolve the merge conflicts (let me know if you have a question on that), then I can push this live!

**Comment by mohatagarvit** [comment]
I have reviewed it from my side. Can you confirm if it's fine now?

**Comment by crweiner** [comment]
The first glance looks fantastic to me!! Thank you so very much for your hard work! You should feel proud of your accomplishments here!! :+1:

many thumbs up

I'm going to have a second glance over things then merge it tonight. You rock and thanks for celebrating Hacktoberfest with me and this little project!!!

**Comment by LorisSecuro** [comment]
@crweiner @mohatagarvit I'd like to point out that the opsdroid reward system is not correlated in any way to Hacktoberfest.
So it probably should not be in the list.
The linked page (https://medium.com/opsdroid/contributor-sticker-packs-738058ceda59) is from over a year ago.
The latest opsdroid blog post regarding Hacktoberfest (https://medium.com/opsdroid/hacktoberfest-2019-ec0e4b4960b9) mentions that (emphasis mine):

> Whenever anyone contributes to opsdroid we send them a sticker pack to say thank you, we do this all year round **and not just for Hacktoberfest**.

@crweiner if you agree, the PR should be reverted, or if you prefer I can try to remove it myself.

**Comment by mohatagarvit** [comment]
Thanks @LorisSecuro for pointing it out. I have made the necessary changes and put this PR #138  for approval. If you still find any fault in this PR's info, please let me know.

**Comment by LorisSecuro** [comment]
@mohatagarvit since this list is about Hacktoberfest and the opsdroid reward is not related, in my opinion it should be removed from the list.
But the project owner is @crweiner, so he'll decide.

**Comment by mohatagarvit** [comment]
@LorisSecuro just to be on the same page. I agree that sticker pack may not be related to Hacktoberfest but the limited edition Hacktoberfest Opsdroid sticker are.

**Comment by LorisSecuro** [comment]
@mohatagarvit good point

**Comment by crweiner** [comment]
Thanks for the extra eyes looking over things @LorisSecuro! 

You're absolutely correct; I want for this list to be Hacktoberfest-related swag only. Because Opsdroid is running a contest for a Hacktoberfest sticker and will be sending that out to people who contribute during October, I am leaning towards keeping them on the list. 

Yes, I see this as a potentially grey area. The fact that they are choosing to celebrate Hacktoberfest by having a custom sticker is having me consider maintaining them here. That's a unique piece of swag for Hacktoberfest even if it's bundled with the "regular" swag too. I'll take a look at #138 to see if that clarifies things a bit. 

Does that sound fair/correct to everyone? If not, please let me know!! Seriously, I have no problems reconsidering if there are enough people who disagree with me! This may be my repo but the content is pretty much in the hands of the public. :smile:

**Comment by LorisSecuro** [comment]
@crweiner I agree with you, the limited edition stickers are related to Hacktoberfest, so it is probably correct to keep them in the list.

**Pull request merged**